src/components/Panels/Jog.js

```diff
@@ -341,6 +341,33 @@ const JogPanel = () => {
       movetoZ = useUiContextFn.getValue("zpos");
     setMoveToTitleZ(T("P75") + movetoZ);
   }, []);
+
+  const toggleUseKeyboard = (e) => {
```

Comment:
I see a difference behavior on menu item hot zone, which may result from a missing class I guess I did not dig in yet
Here my menu item hot zone is working 
![current](https://user-images.githubusercontent.com/8822552/161456109-e5431b64-42fd-4675-b1f6-f27bb89ab291.gif)

On your PR code there is a cold zone on the hot zone - all around  - I did not found why  as I have checked result code and seems identical to current code ...
![PR](https://user-images.githubusercontent.com/8822552/161456162-c9e03f11-df15-4153-8acf-8486ddbd3fac.gif)

Reply:
Ok found it the onclick is not in right place - should be in `<div className="menu-entry" onclick={item.onClick}>` instead of `<div class="menu-panel-item" onclick={item.onClick}>` 
please update it